src/test/java/io/saagie/maven/AbstractSaagieMojoTest.java

```diff
@@ -98,51 +105,65 @@ public void should_load_properties() {
         //Override values
         assertThat(managerProperties.getLogin()).isEqualTo(LOGIN);
         assertThat(managerProperties.getPassword()).isEqualTo(PASSWORD);
+        assertThat(managerProperties.getRealm()).isEqualTo(REALM);
         assertThat(managerProperties.getPlatformId()).isEqualTo(PLATFORM_ID);
         assertThat(managerProperties.getJobName()).isEqualTo(JOB_NAME);
         assertThat(managerProperties.getJobCategory()).isEqualTo(JOB_CATEGORY);
+        assertThat(managerProperties.getDescription()).isEqualTo(DESCRIPTION);
+        assertThat(managerProperties.getReleaseNote()).isEqualTo(RELEASE_NOTE);
     }
 
     @Test
     public void should_generate_url() {
         //Given
         abstractSaagieMojo.loadProperties();
         //When
-        String urlJob = abstractSaagieMojo.generateURLJob(1);
+        String urlJob = abstractSaagieMojo.generateJobURL(1);
         //Then
-        assertThat(urlJob).isEqualTo("https://manager.prod.saagie.io/#/manager/1/job/1");
+        assertThat(urlJob).isEqualTo("https://realm-manager.prod.saagie.io/#/manager/1/job/1");
     }
 
 
     @Test
-    public void should_load_credentials_not_encrypt() throws SecDispatcherException, MojoExecutionException {
+    public void should_load_credentials_not_encrypt() throws MojoExecutionException, MojoFailureException {
```

Comment:
`should_load_unencrypted_credentials` could be better imho

Reply:
done